pretix_eth/static/pretix_eth/question-type.js

```diff
@@ -1,7 +1,143 @@
+"use strict";
+
 /*
  * payment_eth_html_add_question_type_javascript
  */
 
+// Unpkg imports
+const Web3Modal = window.Web3Modal.default;
+const WalletConnectProvider = window.WalletConnectProvider.default;
+const EvmChains = window.EvmChains;
+const Fortmatic = window.Fortmatic;
+
+// Web3modal instance
+let web3Modal;
+
+// Chosen wallet provider given by the dialog window
+let provider;
+
+// Address of the selected account
+let selectedAccount;
+
+function init() {
+  // Tell Web3modal what providers we have available.
+  // Built-in web browser provider (only one can exist as a time)
+  // like MetaMask, Brave or Opera is added automatically by Web3modal
+  const providerOptions = {
+    walletconnect: {
+      package: WalletConnectProvider,
+    },
+
+    fortmatic: {
+      package: Fortmatic,
+    }
```

Comment:
Do we want to use Formatic as a provider? Question for @ligi probably. If so, it looks like we need a key (prob a developer key we obtain from their site?):
![image](https://user-images.githubusercontent.com/4008777/80798933-6f339800-8b9d-11ea-8898-c5f84a5a2333.png)

Reply:
Which providers do we want to use @ligi?

Reply:
at least @walletconnect should be added IMHO. For me fortmatic (or every other provider that needs a key) can be removed IMHO

Reply:
perhaps we should add BurnerConnect for burner wallets